### ISSUE TITLE
WB Indicators produces strange lines with all Nones

### ISSUE THREAD

**Issue by nikicc** [zidarsk8/world_bank_data#1]
When loading `TM.VAL.MRCH.CD.WT` the Data Table on output contains some strange lines containing all `None`:
![screen shot 2017-02-17 at 10 43 25](https://cloud.githubusercontent.com/assets/713026/23060515/50a2ec8a-f4fe-11e6-936a-1cb6c8072221.png)

All other instances use empty strings for missing data. Also Orange then shows `?` in the Data Table to indicate that the value is missing. IMO if this line is not a result of some bug and should even be included in the table it also should use empty strings instead of `None`.

On a side note this caused some problems with Orange3-Text's GeoMap. While the issue should be fixed there https://github.com/biolab/orange3-text/pull/193 it would still be nice to also resolve this here.

**Comment by zidarsk8** [comment]
Yes, this does look weird. I'll see what's causing this and fix it this weekend.

Thanks for the report.
